dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_color_theme.tsx

```diff
@@ -36,12 +36,10 @@ export const ConfigColorTheme = ({
     color: '#FC0505',
   });
 
-  const options = fields
-    .filter((item) => NUMERICAL_FIELDS.includes(item.type))
-    .map((item) => ({
-      ...item,
-      label: item.name,
-    }));
+  const options = fields.map((item) => ({
+    ...item,
+    label: item.name,
```

Comment:
We can add this fix for now.But tickets needs to be on hold till two way implementation is done.